pom.xml

```
@@ -1371,7 +1371,7 @@
     <java.min.version>${compileSource}</java.min.version>
     <!-- Dependencies -->
     <hadoop-two.version>2.7.7</hadoop-two.version>
-    <hadoop-three.version>3.1.1-TDP-0.1.0-SNAPSHOT</hadoop-three.version>
+    <hadoop-three.version>3.1.1-0.0-SNAPSHOT</hadoop-three.version>
```

Find and replace error ?
```suggestion
    <hadoop-three.version>3.1.1-0.0</hadoop-three.version>
```